recipes/Media/README.md

```diff
@@ -0,0 +1,41 @@
+# Media data preparation.
+
+### Contributors :
+- Gaëlle Laperrière, Avignon University, LIA
+- Yannick Estève, Avignon University, LIA
+- Bassam Jabaian, Avignon University, LIA
+- Sahar Ghannay, Paris-Saclay University, CNRS, LISN
+- Antoine Caubrière, Avignon University, LIA
+
+The `parseXMLtoSB.py` script allows to prepare the Media French dataset for experiments. You need both [Media ASR (ELRA-S0272)](https://catalogue.elra.info/en-us/repository/browse/ELRA-S0272/) and [Media SLU (ELRA-E0024)](https://catalogue.elra.info/en-us/repository/browse/ELRA-E0024/) to run the script.
+
+# How to run
+```
+python parseXMLtoSB.py [-w] [-r | -f] (-s | -a) data_folder wav_folder csv_folder
```

Comment:
Can't this be automated as for others recipes? Traditionally the data_prep is all managed by data_prep.py So users to not have to call external scripts.

Reply:
Why? If so, there will be a data preparation (with wav processing) at each experiment, no?

Reply:
This is what happens for all the recipes. Alternatively, in your data prep script (for other recipes this is possible as well) you can specify a save_folder that will store all the processed files, hence if an other experiment pointing towards this dir see that is has already been done, it will be skipped :-)